src/ert/shared/dark_storage/endpoints/server.py

```diff
@@ -0,0 +1,13 @@
+from typing import Mapping, Any
+from fastapi import APIRouter, Depends
+from ert.shared.dark_storage.enkf import LibresFacade, get_res
+
+router = APIRouter(tags=["info"])
+
+
+@router.get("/server/info", response_model=Mapping[str, Any])
+def info(
+    *,
+    res: LibresFacade = Depends(get_res),
+) -> Mapping[str, Any]:
+    return {"name": res.user_config_file}
```

Comment:
Peahaps only return the name of the config file not the whole path to make it more readable in a small tab